src/shell.ts

```diff
@@ -586,10 +586,14 @@ function executeSequence(sequence: Sequence, context: Context): Promise<ExecuteR
 }
 
 function executePipeline(pipeline: Pipeline, context: Context): Promise<ExecuteResult> {
+  const output = executePipelineInner(pipeline.inner, context);
   if (pipeline.negated) {
-    throw new Error("Negated pipelines are not implemented.");
+    return Promise.resolve(output).then((result)=>{
+      result.code = result.code == 0 ? 1 : 0
+      return result
+    })
```

Comment:
Looks about right: https://github.com/denoland/deno_task_shell/blob/fce58e23a8493c054101a214af58c2cf0c80f0ad/src/shell/execute.rs#L335-L341

I think we should only change the code when "kind" !== "exit" though and probably we should return a new object instead of mutating just in case:

```ts
return {
  ...result,
  code: result.code === 0 ? 1 : 0,
};
```

Reply:
Alright change implemented 👍